examples/instance/terraform/plugin.go

```diff
@@ -343,6 +371,16 @@ func (p *plugin) Provision(spec instance.Spec) (*instance.ID, error) {
 
 	id := instance.ID(name)
 
+	storageType, storageName, storageProps, err := FindStorage(&tf)
```

Comment:
@chungers One question - wouldn't the desired behavior be to rename the resource (similar to how the virtual guest resource is renamed)?  Since each instance gets its own tf.json, terraform apply will fail when resource names are repeated (I'm assuming that behavior is not specific to SoftLayer resources, but I don't have experience with others).  Any thoughts?